### PR TITLE
Added guild information onto the engineering website

### PR DESCRIPTION
**Pull request by aaronrussellHO**
Is this pull request a content or a code change? (Please fill in the relevant section and delete the other)

# Code change
I can confirm:
## Accessibility considerations
- [X] Please review the [accessibility checks for layout changes](https://github.com/UKHomeOffice/engineering-guidance-and-standards/blob/main/technical-docs/accessibility/layout-checks.md).

- [X] This change will not change layouts, page structures or anything else that might impact accessibility
- 
# Content change 
- [X] Please review the [accessibility checks for content changes](https://github.com/UKHomeOffice/engineering-guidance-and-standards/blob/main/technical-docs/accessibility/content-checks.md).

I can confirm:
- [X] Content does not include any code or configuration changes (excluding frontmatter information)
- [X] Content meets the content standards
e.g. [Writing a principle](https://engineering.homeoffice.gov.uk/standards/writing-a-principle/) and [Writing a standard](https://engineering.homeoffice.gov.uk/standards/writing-a-standard/)
- [X] Content is suitable to open source, i.e.:
    - Content does not relate to unreleased gov policy
    - Content does not refer to anti-fraud mechanisms
    - Content does not include sensitive business logic

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -9,6 +9,30 @@ tags:
 
 The Home Office Engineering Guidance and Standards site is a body of knowledge for software engineers. It is comprised of principles, patterns and standards that define how we do engineering at the Home Office, and that direct how we want to do engineering in the future. This guidance is created and curated by the Home Office engineering community. The project as a whole is led by the Home Office DDaT Engineering Profession team.
 
+## How we are doing it
+
+Bi weekly the various guilds meet to discuss topics relating to their areas and help to contribute towards the guidance, standards and principles on this site.
+
+The best place to start learning more about the guilds, when they meet and how to join, the best place to start is the Slack.
```
Comment (ThomasWilliamsHO) [inline docs/about.md:16]: "is the Slack" that sounds abit weird to me sorry maybe just "is Slack" ?

Comment (aaronrussellHO) [reply docs/about.md:16]: I think I was going to say Home Office Slack, but then thought, I'm not sure if it's wider than the Home Office. I'll update.

File: docs/about.md [hunk 2]
```diff
@@ -9,6 +9,30 @@ tags:
 
 The Home Office Engineering Guidance and Standards site is a body of knowledge for software engineers. It is comprised of principles, patterns and standards that define how we do engineering at the Home Office, and that direct how we want to do engineering in the future. This guidance is created and curated by the Home Office engineering community. The project as a whole is led by the Home Office DDaT Engineering Profession team.
 
+## How we are doing it
+
+Bi weekly the various guilds meet to discuss topics relating to their areas and help to contribute towards the guidance, standards and principles on this site.
```
Comment (robertdeniszczyc2) [inline docs/about.md:14]:
```suggestion
The guilds meet regularly to discuss topics relating to their areas and contribute towards the guidance, standards and principles.
```

Tweaked wording slightly as not all guilds have the same meeting cadence.

File: docs/about.md [hunk 3]
```diff
@@ -9,6 +9,30 @@ tags:
...
+
+The best place to start learning more about the guilds, when they meet and how to join, the best place to start is Home Office Slack.
+
+On Slack you'll find more information for each guild in their respective channels along with more information on when they meet, and who to contact to join.
+
+### Guild information
```
Comment (robertdeniszczyc2) [inline docs/about.md:20]:
```suggestion
### Guild information
On Home Office Slack you'll find more information for each guild in their respective channels along with when they meet, and who to contact to join.
```

I think we can probably expand this section on here to include at least a one-liner for each guild.

**Review by robertdeniszczyc2 — CHANGES_REQUESTED** [review]

File: docs/about.md (hunk 3 above)
Comment (aaronrussellHO) [reply docs/about.md]: I've added a sentence under each one and made each guild a heading. Hopefully that's a little more info for each!

File: docs/about.md [hunk 4]
```diff
@@ -9,6 +9,55 @@ tags:
...
+Items to be covered to include Open Source, Licensing and Source Code Management.
+
+**Slack**: [#segas-source-management](https://homeoffice.enterprise.slack.com/archives/C02DE9B8M25)
+
+#### Ways of Working
+
+Items to be covered to include Documentation, Tooling and Defining Stories.
+
+**Slack**: [#segas-ways-of-working](https://homeoffice.enterprise.slack.com/archives/C02CLUJ01QX)
+
```
Comment (jeff-horton-ho-sas) [inline docs/about.md:60]:
```suggestion
Build, release and deployment guild
: Topics related to the release and deployment pipeline including CI/CD, Feature Toggling and Artefact Management.

  **Slack**: [#segas-build-release-deployment](https://homeoffice.enterprise.slack.com/archives/C02DR9CU4SC)

Low Code
: A space for to discuss low-code engineering guidance and standards.

  **Slack**: [#segas-low-code](https://homeoffice.enterprise.slack.com/archives/C05MK8UM2F7)

Observability
: Topics relating to the ongoing monitoring of applications and services, including Logging, Analytics and Performance.

  **Slack**: [#segas-observability](https://homeoffice.enterprise.slack.com/archives/C02BJSXJ0QK)

Security
: Articles in this section relate to subjects such as Secure Development, Identity & Access and Vulnerability Scanning.

  **Slack**: [#segas-security](https://homeoffice.enterprise.slack.com/archives/C0282A158FM)

Software Design
: Items to be covered to include Frameworks, Languages, Quality and Coding Standards.

  **Slack**: [#segas-software-design](https://homeoffice.enterprise.slack.com/archives/C02AHL4ML7P)

Source Management
: Items to be covered to include Open Source, Licensing and Source Code Management.

  **Slack**: [#segas-source-management](https://homeoffice.enterprise.slack.com/archives/C02DE9B8M25)

Ways of Working
: Items to be covered to include Documentation, Tooling and Defining Stories.

  **Slack**: [#segas-ways-of-working](https://homeoffice.enterprise.slack.com/archives/C02CLUJ01QX)

```

I think this reads better as a summary list.

File: docs/about.md [hunk 5]
```diff
@@ -9,6 +9,55 @@ tags:
 
 The Home Office Engineering Guidance and Standards site is a body of knowledge for software engineers. It is comprised of principles, patterns and standards that define how we do engineering at the Home Office, and that direct how we want to do engineering in the future. This guidance is created and curated by the Home Office engineering community. The project as a whole is led by the Home Office DDaT Engineering Profession team.
 
+## How we are doing it
```
Comment (edhamiltonHO) [inline docs/about.md:12]: I think this section needs to be combined with the 'How we are managing the site' section further down the page. There is a lot of overlapping information.

File: docs/about.md [hunk 6]
```diff
@@ -9,6 +9,55 @@ tags:
...
+
+The guilds meet regularly to discuss topics relating to their areas and contribute towards the guidance, standards and principles.
```
Comment (edhamiltonHO) [inline docs/about.md:14]: Would be good to introduce what we mean by guilds before this point, it is covered a little bit in the 'why we are doing it' section

File: docs/about.md (hunk 5 above)
Comment (edhamiltonHO) [reply docs/about.md:12]: I would suggest that the easiest way to address this comment, and my other one below regarding introducing guilds, would be to move this section to after the 'why we are doing it' section. As long as that doesn't push the useful guild information too far down the page

Comment (aaronrussellHO) [reply docs/about.md:12]: Yeah, I think that's why I did it. Maybe I'll review the page as a whole and move things around/change things.

File: docs/about.md [hunk 7]
```diff
@@ -9,6 +9,48 @@ tags:
...
+
+### Guild information
+On Home Office Slack you'll find more information for each guild in their respective channels along with when they meet, and who to contact to join.
+
+Build, release and deployment guild
+: Topics related to the release and deployment pipeline including CI/CD, Feature Toggling and Artefact Management.
+
+  **Slack**: [#segas-build-release-deployment](https://homeoffice.enterprise.slack.com/archives/C02DR9CU4SC)
```
Comment (jeff-horton-ho-sas) [inline docs/about.md:22]:
```suggestion
  [**Slack**: #segas-build-release-deployment](https://homeoffice.enterprise.slack.com/archives/C02DR9CU4SC)
  
  [Pages tagged with build, release and deploy](/tags/build-release-and-deploy/)
```

File: docs/about.md [hunk 8]
```diff
@@ -9,6 +9,48 @@ tags:
...
+
+Low Code
+: A space for to discuss low-code engineering guidance and standards.
+
+  **Slack**: [#segas-low-code](https://homeoffice.enterprise.slack.com/archives/C05MK8UM2F7)
```
Comment (jeff-horton-ho-sas) [inline docs/about.md:27]:
```suggestion
  [**Slack**: #segas-low-code](https://homeoffice.enterprise.slack.com/archives/C05MK8UM2F7)
  
  [Pages tagged with low code](/tags/low-code/)
```

File: docs/about.md [hunk 9]
```diff
@@ -9,6 +9,48 @@ tags:
...
+
+Observability
+: Topics relating to the ongoing monitoring of applications and services, including Logging, Analytics and Performance.
+
+  **Slack**: [#segas-observability](https://homeoffice.enterprise.slack.com/archives/C02BJSXJ0QK)
```
Comment (jeff-horton-ho-sas) [inline docs/about.md:32]:
```suggestion
  [**Slack**: #segas-observability](https://homeoffice.enterprise.slack.com/archives/C02BJSXJ0QK)
  
  [Pages tagged with observability](/tags/observability/)
```

File: docs/about.md [hunk 10]
```diff
@@ -9,6 +9,48 @@ tags:
...
+
+Security
+: Articles in this section relate to subjects such as Secure Development, Identity & Access and Vulnerability Scanning.
+
+  **Slack**: [#segas-security](https://homeoffice.enterprise.slack.com/archives/C0282A158FM)
```
Comment (jeff-horton-ho-sas) [inline docs/about.md:37]:
```suggestion
  [**Slack**: #segas-security](https://homeoffice.enterprise.slack.com/archives/C0282A158FM)
  
  [Pages tagged with security](/tags/security/)]
```

File: docs/about.md [hunk 11]
```diff
@@ -9,6 +9,48 @@ tags:
...
+
+Software Design
+: Items to be covered to include Frameworks, Languages, Quality and Coding Standards.
+
+  **Slack**: [#segas-software-design](https://homeoffice.enterprise.slack.com/archives/C02AHL4ML7P)
```
Comment (jeff-horton-ho-sas) [inline docs/about.md:42]:
```suggestion
  [**Slack**: #segas-software-design](https://homeoffice.enterprise.slack.com/archives/C02AHL4ML7P)
  
  [Pages tagged with software design](/tags/software-design/)
```

File: docs/about.md [hunk 12]
```diff
@@ -9,6 +9,48 @@ tags:
...
+
+Source Management
+: Items to be covered to include Open Source, Licensing and Source Code Management.
+
+  **Slack**: [#segas-source-management](https://homeoffice.enterprise.slack.com/archives/C02DE9B8M25)
```
Comment (jeff-horton-ho-sas) [inline docs/about.md:47]:
```suggestion
  [**Slack**: #segas-source-management](https://homeoffice.enterprise.slack.com/archives/C02DE9B8M25)
  
  [Pages tagged with source management](/tags/source-management/)
```

File: docs/about.md [hunk 13]
```diff
@@ -9,6 +9,48 @@ tags:
...
+
+Ways of Working
+: Items to be covered to include Documentation, Tooling and Defining Stories.
+
+  **Slack**: [#segas-ways-of-working](https://homeoffice.enterprise.slack.com/archives/C02CLUJ01QX)
```
Comment (jeff-horton-ho-sas) [inline docs/about.md:52]:
```suggestion
  [**Slack**: #segas-ways-of-working](https://homeoffice.enterprise.slack.com/archives/C02CLUJ01QX)
  
  [Pages tagged with ways of working](/tags/ways-of-working/)
```

**Review by jeff-horton-ho-sas — COMMENTED** [review]
* Add links to the tag page for each guild
* Move the `Slack:` into the link text so that the context is clear when screen readers list all the links on a page.

File: docs/about.md (hunk 5 above)
Comment (aaronrussellHO) [reply docs/about.md:12]: I've moved some stuff around. I've left the info about how we manage the site as that feels different to me. @edhamiltonHO

**Review by jeff-horton-ho-sas — APPROVED** [review]

**Review by edhamiltonHO — APPROVED** [review]

**Review by robertdeniszczyc2 — APPROVED** [review]

**Pull request merged**